app/views/step_by_step_pages/internal_change_notes.html.erb

```diff
@@ -28,6 +28,23 @@
 <% displayed_edition_number = nil %>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <%= form_for(@internal_change_note, url: step_by_step_page_internal_change_notes_path) do |form| %>
```

Comment:
Can this change be fixed the commit [Move internal notes to aside 1/3"](3dfb1c621d2613b0c12fde25b3b641d39f453485)

Reply:
Yep, will merge a few commits as we pivoted on the approach.